cost/low_service_usage/low_service_usage.pt

```diff
@@ -293,7 +293,7 @@ end
 escalation "esc_budget_alert" do
   automatic true
   label "Send Email"
-  description "Send incident email"
+  description "Send Incident email"
```

Comment:
Can we revert it to the "incident" as it is in lower case in all other policies?